src/components/Toolbar.tsx

```diff
@@ -70,6 +73,32 @@ async function showGitOperationDialog(
   }
 }
 
+async function showTagListDialog(
+  model: IGitExtension,
+  repo: IToolbarState['repository']
+): Promise<void> {
+  const title = `Tags`;
+  let tagsValue = await showDialog({
+    title: title,
+    body: new GitTagDialog(model, repo),
+    buttons: [Dialog.okButton({ label: 'OK' })]
+  });
+  while (!tagsValue.button.accept) {
+    tagsValue = await showDialog({
+      title: title,
+      body: new GitTagDialog(model, repo),
+      buttons: [Dialog.okButton({ label: 'DISMISS' })]
+    });
+  }
+  if (tagsValue.button.accept) {
+    tagsValue = await showDialog({
```

Comment:
```suggestion
    await showDialog({
```

You don't do anything with the returned value.

Reply:
tagsValue is used to trigger the checkout api